packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -51,7 +59,9 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
     onChange,
     onDragStart,
     onDragEnd,
-    getA11ValueText = (value) => `${value.toFixed(0)}px`,
+    getA11ValueText = (value) =>
+      value === UNMEASURED ? 'Unknown' : `${value.toFixed(0)}px`,
```

Comment:
Let's just return undefined instead of 'Unknown' - since we don't want a default un localized screen reader message.

Also in the documentation there is no mention that a value of `-1` is considered unmeasured - let's reflect that. I think it should be enough to add a11y text to the story example.

Reply:
I actually wanted to create a follow up ticket on updating this value with the actual element size when the handle gets focused. But until then, I agree undefined will work better.